README.md

```diff
@@ -1,2 +1,114 @@
+<p align="right">
+  <img width="25%" height="25%"src="./images/klepsydra_logo.jpg">
+</p>
+
 # kpsr-queue-benchmarks
 Klepsydra queue benchmarks
+
+# Installation Instructions
+
+## System dependencies
+
+* Ubuntu 14.04 or above
+* Cmake 3.5.1 or above
+* gcc for C++11 5.4.0 or above.
+* Google Test (https://github.com/google/googletest)
+* Google Benchmark (https://github.com/google/benchmark)
+
+## Installation
+
+Given `$KLEPSYDRA_HOME`, for example `~/klepsydra`
+
+```bash
+cd $KLEPSYDRA_HOME
+git clone https://github.com/klepsydra-technologies/kpsr-queue-benchmarks
+cd kpsr-queue-benchmarks
+mkdir build
+cd build
+cmake ..
+make
+```
+## Run benchmarks
+
+Benchmarks can be run individually by the following command:
+
+```bash
+cd kpsr-queue-benchmarks/build
+./bin/queue_benchmarks_<name_of_the_benchmark>
+```
+Or run all benchmarks one after the other with the following command:
+
+```bash
+cd kpsr-queue-benchmarks/build
+for benchmark in *; do ./$benchmark; done
```

Comment:
The command must be run inside the bin folder, so there are 2 options:

1. Change the first line to: **cd kpsr-queue-benchmarks/build/bin**
2. Or, change the second line to: **for benchmark in ./bin/*; do $benchmark; done**